src/_guides/language/analysis-options.md

```diff
@@ -204,22 +204,64 @@ For example, some rules are more appropriate for library packages
 and others are designed for Flutter apps.
 Note that linter rules can have false positives, unlike static analysis.
 
+### Enabling Dart team recommended rules: lints {#lints}
+
+The Dart team provides two sets of recommended linter rules in the [lints package][]:
+
+Core rules
+: Help identify critical issues that are likely to lead to problems
+  when running or consuming Dart code.
+  All code should pass these linter rules 
+  as Dart packages uploaded to [pub.dev]({{site.pub}})
+  are [scored]({{site.pub}}/help/scoring) with the these.
+
+Recommended rules
+: Help identify additional issues
+  that may lead to problems when running or consuming Dart code,
+  and enforce a single, idiomatic style and format.
+  We recommend that all Dart code use these rules,
+  which are a superset of the core rules.
+
```

Comment:
```suggestion

{{site.alert.tip}}
  If you're working on Flutter code,
  then instead of directly using `lints`, use [`flutter_lints`][],
  which provides a superset of the recommended rules.
{{site.alert.end}}

```